pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -188,6 +189,23 @@ static __always_inline int handle_retransmit(struct sock *sk, int segs) {
     return 0;
 }
 
+static __always_inline int handle_retransmit_retrans_out(struct sock *sk) {
+    conn_tuple_t t = {};
+    u64 zero = 0;
+
+    if (!read_conn_tuple(&t, sk, zero, CONN_TYPE_TCP)) {
+        return 0;
+    }
+
+    struct tcp_sock *tcp_sk = (struct tcp_sock *)&sk;
+    u32 * retrans_out = &(tcp_sk->retrans_out);
```

Comment:
You are probably going to have a problem loading this. Use `bpf_probe_read` here to read the value into a local var.